filter/src/abpvn_general.txt

```diff
@@ -186,6 +186,7 @@
 ||quaivatdienanh.com/*banner$image
 ||quangcaosanpham.com/images/$third-party
 ||r2.plvb.xyz^*.gif$image,domain=xoilaczf.tv
+||raovat.tuoitre.vn^$xhr,1p
```

Comment:
Should we use `1p` or `domain=tuoitre.vn` here? Is there any compatibility issue across extensions, i.e. uBO, AdGuard and ABP?

Reply:
$xhr should be $xmlhttprequest and 1p should be $domain=tuoitre.vn for compare with all adblocker